src/Activity.js

```diff
@@ -5,24 +5,54 @@ class Activity {
     this.activityData = activityData;
   };
 
-  getMinutesActiveByDate(date) {
-    const minutesActive = this.usersActivity
-      .find(data => data.date === date);
+  getDailyActivityByDate(type, date) {
+    const value = this.usersActivity.find(data => data.date === date);
 
-    return minutesActive.minutesActive;
+    if (value) {
+      return value[type];
+    } else {
+      return 'This date could not be found.';
+    };
+
+  };
+
+  getDailyMilesByDate(date) {
+    const totalMiles = this.usersActivity.reduce((total, data) => {
+      const mileInFeet = 5280;
+      if (data.date === date) {
+        total = this.currentUser.strideLength * data.numSteps / mileInFeet
```

Comment:
semicolon here

Reply:
Awesome job on Activity! Excited to see these charts :)

Reply:
Will add on the next push up